modmesh/pylibmgr.py

```diff
@@ -27,10 +27,35 @@
 import sys
 import os
 import time
+import importlib.abc
+import importlib.machinery
 
 lib_path = {}
 
 
+class mm_path_finder(importlib.abc.MetaPathFinder):
```

Comment:
Why this class is not using `CamelCase`?

Reply:
I have changed it at the latest commit.